config/database.yml

```diff
@@ -0,0 +1,12 @@
+default: &default
+  adapter: postgresql
+  encoding: utf8
+  pool: 5
+  timeout: 5000
+  username: katherinafatiguso
```

Comment:
you should use something like toyrobotuser as I don't want to create a user on my machine called katherinaf..